students/HarveyHerela/lesson03/list_lab.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+# Starting list of fruits
+my_fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+print(my_fruits)
+
+# Query the user for another fruit
+response = input("Fruit to add: ")
+my_fruits.append(response.title())
+print(my_fruits)
+
+# Ask for a fruit index
+response = 'a'
+while not response.isdigit():
+    response = input("Index of a fruit to display: ")
+    # Verify it's an int
+    if response.isdigit():
+        fruit_index = int(response)
+        # Verify it's in range
+        if fruit_index > len(my_fruits):
+            response = 'a'
+    
+print(fruit_index, my_fruits[fruit_index - 1])
+
+# Query the user for another fruit: +
+response = input("Fruit to add: ")
+my_fruits = [response.title()] + my_fruits
+print(my_fruits)
+
+# Query the user for another fruit: insert
+response = input("Fruit to add: ")
+my_fruits.insert(0, response.title())
+print(my_fruits)
+
+# Display all fruits that start with a p using a for loop
+print("Displaying all fruits that start with the letter 'p'")
+for fruit in my_fruits:
+    if fruit[0] == 'P':
+        print(fruit, end=' ')
+print('\n')
+
+
+# Start of series 2
+print("Start of series 2")
+# Display the list.
+print(my_fruits)
+
+# Remove the last fruit from the list.
+# Display the list.
+print("Removing the last fruit")
+my_fruits.pop(len(my_fruits)-1)
+print(my_fruits)
+
+# Ask the user for a fruit to delete, find it and delete it.
+# (Bonus: Multiply the list times two. Keep asking until a match is found. Once found, delete all occurrences.)
+my_fruits = my_fruits * 2
+num = 0
+while num == 0:
```

Comment:
See if you can use the `in` keyword here to make this more concise e.g. `while response in my_fruits:` ...